tests/phpunit/Fixtures/SimpleMutation.php

```diff
@@ -4,10 +4,12 @@
 
 namespace Infection\Tests\Fixtures;
 
+use Infection\MutatedNode;
+use Infection\Mutation\Mutation;
 use Infection\Mutator\Mutator;
 use PhpParser\Node;
 
-class SimpleMutation
+class SimpleMutation extends Mutation
```

Comment:
Might be worth looking in another PR if `SimpleMutation` is still necessary? I feel like a test factory for `Mutator` might be more appropriate than this class... if necessary at all